infra/gcp/tests/gke.tf

```diff
@@ -141,6 +145,9 @@ module "prow_build" {
       auto_repair        = true
       auto_upgrade       = true
       service_account    = google_service_account.gke_nodes.email
+      oauth_scopes       = [
+        "https://www.googleapis.com/auth/cloud-platform"
```

Comment:
This scope is already present. 

```
 REDACTED  MCW0CDP3YY  ~  Desktop  Git  kn-hack   rename-master-function  $   gcloud container clusters describe prow-build --region us-central1 --project knative-tests --format json | jq .nodePools[1]
{
  "autoscaling": {
    "enabled": true,
    "maxNodeCount": 4
  },
  "config": {
    "diskSizeGb": 100,
    "diskType": "pd-ssd",
    "imageType": "COS_CONTAINERD",
    "labels": {
      "cluster_name": "prow-build",
      "environment": "production",
      "node_pool": "testing-pool-v1",
      "type": "testing"
    },
    "localSsdCount": 1,
    "machineType": "n2d-standard-16",
    "metadata": {
      "cluster_name": "prow-build",
      "disable-legacy-endpoints": "true",
      "node_pool": "testing-pool-v1"
    },
    "oauthScopes": [
      "https://www.googleapis.com/auth/cloud-platform"
    ],
    "serviceAccount": "gke-nodes@knative-tests.iam.gserviceaccount.com",
    "shieldedInstanceConfig": {
      "enableIntegrityMonitoring": true,
      "enableSecureBoot": true
    },
    "tags": [
      "gke-prow-build",
      "gke-prow-build-testing-pool-v1"
    ],
    "workloadMetadataConfig": {
      "mode": "GKE_METADATA"
    }
  },
  "initialNodeCount": 1,
  "instanceGroupUrls": [
    "https://www.googleapis.com/compute/v1/projects/knative-tests/zones/us-central1-c/instanceGroupManagers/gke-prow-build-testing-pool-v1-69a0049f-grp",
    "https://www.googleapis.com/compute/v1/projects/knative-tests/zones/us-central1-f/instanceGroupManagers/gke-prow-build-testing-pool-v1-2ca0560d-grp"
  ],
  "locations": [
    "us-central1-c",
    "us-central1-f"
  ],
  "management": {
    "autoRepair": true,
    "autoUpgrade": true
  },
  "maxPodsConstraint": {
    "maxPodsPerNode": "110"
  },
  "name": "testing-pool-v1",
  "networkConfig": {
    "podIpv4CidrBlock": "10.250.64.0/20",
    "podRange": "prow-build-pods"
  },
  "podIpv4CidrSize": 24,
  "selfLink": "https://container.googleapis.com/v1/projects/knative-tests/locations/us-central1/clusters/prow-build/nodePools/testing-pool-v1",
  "status": "RUNNING",
  "upgradeSettings": {
    "maxSurge": 1
  },
  "version": "1.22.8-gke.200"
}
```

Also, e2 machine types don't support local ssds.

Reply:
I see. Then it's very weird. I logged into a boskos janitor pod and ran `gcloud auth activate-service-account --key-file=/etc/test-account/service-account.json`, it just got stuck for a while and then said `Killed`: 

```
root@boskos-janitor-5b7788867d-hl2hl:/# gcloud auth activate-service-account --key-file=/etc/test-account/service-account.json
Killed
```

Reply:
Reverted those changes. The other changes in this PR might still be desirable.